versions.tf

```
@@ -2,10 +2,11 @@ terraform {
   required_version = ">= 0.12.9"
 
   required_providers {
-    aws      = ">= 2.38.0"
-    local    = ">= 1.2"
-    null     = ">= 2.1"
-    template = ">= 2.1"
-    random   = ">= 2.1"
+    aws        = ">= 2.38.0"
+    local      = ">= 1.2"
+    null       = ">= 2.1"
+    template   = ">= 2.1"
+    random     = ">= 2.1"
+    kubernetes = "~> 1.10"
```

We should only use `=>` here to avoid conflicts with users and other module definitions. 2.0 might be compatible with our resource. We can't know yet.

Our minimum compatible version is 1.6.2, the first correctly tagged with TF 0.12 support.
```suggestion
    kubernetes = "=> 1.6.2"
```